chaosk8s/pod/actions.py

```diff
@@ -46,21 +47,111 @@ def terminate_pods(label_selector: str = None, name_pattern: str = None,
     be used as the grace period (in seconds) to terminate the pods.
     Otherwise, the default pod's grace period will be used.
     """
+
+    api = create_k8s_api_client(secrets)
+    v1 = client.CoreV1Api(api)
+
+    pods = _select_pods(v1, label_selector, name_pattern,
+                        all, rand, mode, qty, ns, order)
+
+    body = client.V1DeleteOptions()
+    if grace_period >= 0:
+        body = client.V1DeleteOptions(grace_period_seconds=grace_period)
+
+    for p in pods:
+        v1.delete_namespaced_pod(p.metadata.name, ns, body=body)
+
+
+def exec_in_pods(label_selector: str = None, name_pattern: str = None,
+                 all: bool = False, rand: bool = False,
+                 mode: str = "fixed", qty: int = 1,
+                 ns: str = "default", order: str = "alphabetic",
+                 secrets: Secrets = None, cmdstr: str = None,
+                 container_name: str = None):
+    """
+    Exceute a commnand in the specified pod's container.
```

Comment:
typo

Reply:
I have added more tests to improve the coverage.